lib/candig-data-portal/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:14.16.0-alpine
+
+LABEL Maintainer="CanDIG Project"
+
+RUN apk add --no-cache git
+RUN apk add --no-cache curl
+
+COPY candig-data-portal /app/candig-data-portal
+
+WORKDIR /app/candig-data-portal
```

Comment:
WORKDIR should be the first step this Dockerfile as it will not change. This will reduce the need to re-cache container steps at build time